anime_downloader/sites/animesvostfr.py

```diff
@@ -0,0 +1,247 @@
+                option = select.find('option', {'selected': True})
+            if option:
+                episodeid = option['episodeid']
+                episodenum = option.text
+            
+            logger.debug( "_get_sources  filmid: %s episodeid: %s episodenum: %s", str(filmid), str(episodeid), str(episodenum) )
+            
+            
+            if not filmid:
+                filmid = episodeid
+            
+            episodeurl = None
+            if filmid:
+                episodeurl = helpers.get(self.link_stream_url, params={'server' : 'rapidvideo', 'filmId' : filmid}, cf=True).text.strip()
+                
+                if not episodeurl or episodeurl == '':
+                    episodeurl = helpers.get(self.link_stream_url, params={'server' : 'photoss', 'filmId' : filmid}, cf=True).text.strip()
+                    
+            else:
+                raise NotFoundError
+                
+            logger.debug( "_get_sources url:%s", episodeurl )
+            
+            
+            
+            
+            #https://youtubedownloader.cx/v/y0661ue6-8jzj34
+            #https://youtubedownloader.cx/api/source/y0661ue6-8jzj34
+            
+            
+            #https://play.comedyshow.to/embedplay/4a2faf54b99e8d1152e598b24916d0d9
+            #<iframe src="https://www.fembed.net/v/1ekm5ij-3l2rp8p"
+            #https://feurl.com/api/source/1ekm5ij-3l2rp8p
+            # POST r: https://play.comedyshow.to
+            #d: feurl.com
+            
+            episodeurl1 = episodeurl          
+            episodeurlapi1 = episodeurl1.replace('/v/', '/api/source/' )
+            
+            episodeurl2 = ''
+            episodeurlapi2 = ''
+            
+            soup = helpers.soupify(helpers.get(episodeurl, cf=True))
+            #logger.debug( "results: {}".format(soup) )
+            
+            iframe = soup.select_one("iframe")
+            if iframe:
+                episodeurl2 = iframe['src']
+                
+                logger.debug( "_get_sources episodeurl2:%s", episodeurl2 )
+                
+                episodeurlapi2 = episodeurl2.replace('/v/', '/api/source/' )
+                
+            
+            logger.debug( "_get_sources episodeurlapi1:%s", episodeurlapi1 )    
+            logger.debug( "_get_sources episodeurlapi2:%s", episodeurlapi2 )   
+            
+            #data = helpers.post(episodeurlapi, params={'r' : '', 'd' : 'youtubedownloader.cx' }, referer=episodeurl, cf=True).json()
+            #data = helpers.post(episodeurlapi, params={'r' : '', 'd' : '' }, referer=episodeurl).json()
+            #data = helpers.post(episodeurlapi, params={'r' : '' }, referer=episodeurl).json()
+            
+            data = None
+            if episodeurlapi2:
+                data = helpers.post(episodeurlapi2, referer=episodeurl2).json()
+            elif episodeurlapi1:
+                data = helpers.post(episodeurlapi1, referer=episodeurl1).json()
```

Comment:
getting a 404 error here when doing 
`anime -ll DEBUG dl "neverland" --provider animesvostfr -c 1 -e 1`
Looks like file was deleted, please make a graceful exit when that happens

Reply:
OK i have to check it